packages/asc-ui/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useImperativeHandle } from 'react'
+import useActionOnEscape from '../../utils/hooks/useActionOnEscape'
+import useFocusOnRender from '../../utils/hooks/useFocusOnRender'
+import TextAreaContext from './TextAreaMethodsContext'
+import TextAreaStyle from './TextAreaStyle'
+
+export interface TextAreaMethods
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  onWatchValue?: (value: string) => void
```

Comment:
I think this one can be removed, right? not used anywhere..

Reply:
Good catch.

I don't think it's being used in the `Input`-component either right?